bedrock/firefox/templates/firefox/accounts-2018.html

```diff
@@ -0,0 +1,234 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+  # License, v. 2.0. If a copy of the MPL was not distributed with this
+  # file, You can obtain one at http://mozilla.org/MPL/2.0/. #}
+
+{% from "macros-protocol.html" import card, call_out, feature_card with context %}
+{% from "macros.html" import fxa_email_form, google_play_button, send_to_device, fxa_create_account_button with context %}
+
+{% extends "base-protocol.html" %}
+
+{% block page_title %}{{ _('Get a Firefox Account – Keep your data private, safe and synced') }}{% endblock %}
+{% block page_desc %}{{ _('Securely sync your passwords, bookmarks and tabs across all your devices. Get a Firefox Account now – One log-in – Power and privacy everywhere.') }}{% endblock %}
+
+{% block page_css %}
+ {{ css_bundle('firefox_accounts_2018') }}
+{% endblock %}
+
+{% block content %}
+
+<section class="mzp-c-hero mzp-has-image mzp-t-firefox mzp-t-product-firefox">
+  <div class="mzp-l-content">
+    <div class="mzp-c-hero-body">
+      <h1 class="mzp-c-hero-title">{{ _('One log-in. Power and privacy everywhere.') }}</h1>
+      <div id="hero-cta" class="mzp-c-hero-cta">
+        <div class="show-fxa-supported-signed-out">
+          <a href="https://accounts.firefox.com/signup?service=sync&context=fx_desktop_v3&entrypoint=mozilla.org-button-light&utm_content=%2Ffirefox%2Faccounts%2F&utm_campaign=Create an Account" class="button account-benefits" data-button-name="Create account" data-link-type="button" data-cta-position="primary cta" id="features-hero-account">
```

Comment:
Can we use the `fxa_create_account_button` macro here? It's possible to configure the button label

Reply:
I had forgotten about this road bump until I started playing with the macro again-- 
<img width="368" alt="screen shot 2018-11-30 at 11 07 29 am" src="https://user-images.githubusercontent.com/30483988/49309468-37985c00-f490-11e8-8456-66e9515e2135.png">
The macro automatically displays the mobile buttons when logged in & in firefox, but we wanted to do a custom modal with the QR & buttons instead so I opted for this handwritten option.

Reply:
👍